sdk/core/azure-core-tracing-opentelemetry/src/samples/ListKeyVaultSecretsTracingSample.md

```diff
@@ -23,7 +23,7 @@ Sample uses **[opentelemetry-sdk][opentelemetry_sdk]** as implementation package
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-secrets</artifactId>
-    <version>4.0.1</version>
+    <version>4.1.0-beta.1</version>
```

Comment:
I think this should be marked as a dependency version and we should update how these samples are injected into the README to ensure they compile and use the right version.